internal/distro/rhel9/distro.go

```diff
@@ -273,6 +273,13 @@ func newDistro(name string, major, minor int) distro.Distro {
 		},
 	}
 
+	azureAarch64Platform := &platform.Aarch64{
```

Comment:
nitpick: I wonder whether we should not be adding the Azure-specific platform definitions to the `azure.go` as the rest of Azure-specific things... 🤔